### PR TITLE
Gable

### PR DESCRIPTION
**Pull request by Joost-s-d**
Gable is a lending protocol on Radix DLT. V1 of the protocol stores XRD and XRD derived liquid staking units (LSU) in the contract. LSU's are derived from XRD, and are 1 on 1 exchangeable - thats why LSU's are valued on the same price as XRD.

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):  

Gable

##### Twitter Link: 

https://twitter.com/Gable_Finance

##### List of audit links if any: 

https://hacken.io/audits/gable-finance/

##### Website Link: 

https://gable.finance

##### Logo (High resolution, will be shown with rounded borders):

https://gable.finance/gable.png

##### Current TVL:

700k USD

##### Treasury Addresses (if the protocol has treasury)

account_rdx168nhk2f07xg3vqgp6ntlel08f8w3cy4mhq4nwt7gky33z84sxncljl

##### Chain:

Radix DLT

##### Short Description (to be shown on DefiLlama):

Gable is a lending protocol, built on Radix DLT.

##### Token address and ticker if any:

resource_rdx1tknu3dqlkysz9lt08s7spuvllz3kk2k2yccslfpdk73t4lnznl9jck: GAB

##### Category (full list at https://defillama.com/categories) *Please choose only one:

2, Lending

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):

-

##### forkedFrom (Does your project originate from another project):

-

##### methodology (what is being counted as tvl, how is tvl being calculated):

XRD: Liquidity in the contract used for flash loans
LSU: Locked in the contract used for staking and interest earnings

##### Github org/user (Optional, if your code is open source, we can track activity):

https://github.com/gable-finance/gable

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/gable exports TVL: 
        
 
 
    radixdlt                  473.57 k
    
    total                    473.57 k

File: projects/gable/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { get } = require('../helper/http')
+
+async function tvl(_, _b, _cb, { api, }) {
+  const response = await get('https://www.sundaefinance.net:8443/api/tvl_data');
```
Comment (g1nt0ki) [inline projects/gable/index.js:4]: can you please fetch this info on chain? we have basic support for making calls to rdx chain: https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/helper/chain/radixdlt.js

**Comment by llamatester** [comment]
The adapter at projects/gable exports TVL: 
        
 
 
    radixdlt                  505.2554203
    
    total                    505.2554203

File: projects/gable/index.js (hunk 1 above)
Comment (Joost-s-d) [reply projects/gable/index.js]: Hi @g1nt0ki, I have adjusted this accordingly.

The problem here is that the contract stores liquid staking units (LSU's). These LSU's are not tradable, they are redeemable for XRD on a 1:1 ratio. So essentially, the value of these LSU are equal to the value of XRD, but there is no price information available for this as they are not being traded.

**Comment by g1nt0ki** [comment]
@Joost-s-d can you also fill in the project details. I will think of a solution for the lSU

**Comment by llamatester** [comment]
The adapter at projects/gable exports TVL: 
        
 
 
    radixdlt                  770.33 k
    
    total                    770.33 k

**Comment by Joost-s-d** [comment]
@g1nt0ki thank you for looking into this. I have filled in the form.

**Comment by g1nt0ki** [comment]
hi @Joost-s-d on second thought it is better if we make two sub listings one for flashloans and other for Liquid Staking/yield

**Comment by Joost-s-d** [comment]
Hi @g1nt0ki sure, would that mean that the adapter will be split into two adapters inside the same folder? I see that the pull request is closed - is this still applicable?

Additionally, I see that there is a disclaimer on the TVL stating that the protocol is under liquid staking category, so the TVL is subtracted. This might be applicable to other chains; however, as LSU's are native to Radix, they are inherently not counted as TVL. However, from the moment these LSU's are locked into Gable and used for yield farming, they should be counted as TVL.

**Comment by g1nt0ki** [comment]
hi @Joost-s-d ended up decided to leave it as single protocol. 

regarding not counting as tvl, we dont count yeild protocols as tvl either (users can enable it in filter), example, convex or yearn on ethereum

**Pull request merged**